ui/gtk/main_window_button.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class MainWindowButton(Enum):
```

Comment:
You can probably move this enum to `main_window.py`, that would eliminate this file which doesn't add much, and keep everything in a single module.

Reply:
Can't because the pages will include main_window and main_window will include pages.